delete_module.php

```diff
@@ -50,24 +50,15 @@
 
     echo $OUTPUT->header();
 
-    $cm = new stdClass();
-    $cmsdata = get_all_cmdelete_adhoctasks_data(array($cmid));
-    foreach ($cmsdata as $cmtaskid => $taskdata) {
-        $taskdata = get_cms_infos($taskdata);
-        foreach ($taskdata as $coursemoduleid => $cmdata) {
-            if ($coursemoduleid == $cmid) {
-                $cm = $cmdata;
-                break;
-            }
-        }
-    }
-    $deleteoutput = force_delete_module_data($cm, $taskid, true);
+    $minimumfaildelay = intval(get_config('tool_fix_delete_modules', 'minimumfaildelay'));
```

Comment:
Can this be passed down to reporter constructor?

Reply:
will fix this

Reply:
For the review convenience please feel free to resolve the issue once it's actually fixed. Otherwise, it's a bit hard to track review notes